llvm/lib/Target/WebAssembly/WebAssemblyCFGStackify.cpp

```diff
@@ -1850,13 +1850,12 @@ bool WebAssemblyCFGStackify::fixCallUnwindMismatches(MachineFunction &MF) {
 
       // If the EH pad on the stack top is where this instruction should unwind
       // next, we're good.
-      MachineBasicBlock *UnwindDest = getFakeCallerBlock(MF);
+      MachineBasicBlock *UnwindDest = nullptr;
```

Comment:
Drive-by NFC fix. Fake caller block is used not here but somewhere below, and this part of the code cannot be entered when there is no EH pad successor:
https://github.com/llvm/llvm-project/blob/ab87206c4b95aa0b5047facffb5f78f7fe6ac269/llvm/lib/Target/WebAssembly/WebAssemblyCFGStackify.cpp#L1846-L1848
which will be assigned to `UnwindDest` below, so this has no meaning other than creating an unnecessary BB.